Tests/conf.json

```diff
@@ -2884,6 +2910,9 @@
             "integrations": "Workday",
             "playbookID": "Workday - Test",
             "fromversion": "5.0.0"
+        },
+        {
+            "playbookID": "Darktrace Test Playbook"
```

Comment:
Please remove

Reply:
Removed -- please let me know if this was resolved correctly

Reply:
It was , great.